locale/uk_UA/editor.xml

```diff
@@ -2,15 +2,153 @@
 <!DOCTYPE locale SYSTEM "../../dtd/locale.dtd">
 
 <!--
-  * locale/uk_UA/editor.xml
+  * locale/en_US/editor.xml
   *
-  * Copyright (c) 2013-2018 Simon Fraser University
+  * Copyright (c) 2014-2018 Simon Fraser University Library
   * Copyright (c) 2003-2018 John Willinsky
   * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
-  * Localization information: https://pkp.sfu.ca/wiki/index.php?title=OJS:_Ukrainian_(uk_UA)
```

Comment:
Please leave the links to the wiki in place -- that's where we provide credit to contributors to the translation.